TAC/Meetings/2022/2022-02-24/CCC_TAC_Minutes-2022-02-24.md

```diff
@@ -0,0 +1,193 @@
+
+1. **[Stephen Walli, Dan Middleton, Aeva Black]** Develop a CoC escalation process, to be presented to TAC and governing board.
+    * Dan Middleton has opened a PR for review
+
+1. **[Brian Warner]** Add LFX security code scanning for projects as a topic for a future meeting.
+    * To be discussed in main agenda as a possible TAC tech talk.
+
+### Completed Action Items
+
+1. **[ALL]** Review, comment and/or correct text in [Common Terminology Document](https://docs.google.com/document/d/1xZ6IX0w0jaWDbLMFNAybTF3FpLnQ5TJ98nzIWbsbFnY/edit). (To be moved to main agenda)
+
+
+1. **[Dan Middleton]** [PR #92](https://github.com/confidential-computing/governance/pull/92) opened to develop a CoC escalation process.
+
+
+### Project overviews
+
+* Veraison
+    * Dave Thaler (DT) noted that Veraison was approved by the TAC, and is ready for next steps (technical charter, IP transfer, board approval).
+* Enarx
+    * DT noted that Enarx is ready for its annual review.
+
+1. **[Brian, Dave]** Schedule Veraison for Board review, tech charter development, and IP transfer.
+
+### Open Enclave SDK - annual project review 
+
+Radhika Jandhyala (RJ) reviewed the [Open Enclave SDK annual project review document](https://github.com/confidential-computing/governance/tree/main/TAC/Meetings/2022/2022-02-24). She noted that the communication channels have changed, noting that the Matrix server has been deprecated. She also noted that some dependencies have changed.
+
+DT asked for clarification which SGX SDK is represented in the dependencies, as there are different versions for Windows and Linux. RJ noted that this dependency is specific to the SGX SDK for Linux.
+
+RJ noted that Linux packages have been uploaded to various package repositories and GitHub.
+
+She noted that Open Enclave SDK has a quarterly release cadence. Releases are also made when CVEs are fixed. She noted that the project averages six commits weekly. The committers are making progress towards Inclusive Open Source Communities orientation, and noted that this is being tracked in a github issue.
+
+RJ reviewed intentional work by the project to help diversify the organizations involved. She noted that that OE SDK has not yet published a list of adopters and is not yet ready to request Graduation stage.
+
+RJ next reviewed budget requests. She noted that OE SDK is using Zoom and had a license scan in the fall. She requested an updated license scan once the project has had time to remediate some issues. She noted this is likely the only budget request.
+
+DT requested RJ's opinion whether the project could be ready for Graduation status within the next year. She noted that it could be longer. DT noted that the Graduated lifecycle text has never been finalized. Dan Middleton (DM) asked if this should be a focus for an upcoming meeting.
```

Comment:
```suggestion
DT requested RJ's opinion whether the project could be ready for Graduation status within the next year. She noted that it could be longer. DT noted that the Graduation lifecycle text has never been finalized. Dan Middleton (DM) asked if this should be a focus for an upcoming meeting.
```